Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_006.GitHubPat.ps1

```diff
@@ -18,4 +20,12 @@ https://avatars0.githubusercontent.com/u/16199012?u=4196919fb12cc4aa8ec0159a746c
 https://github.com/microsoft/sarif-pattern-matcher/tree/7af498925c89a9e8c07fb8a65223e89d513d4eea
 
 # This should trigger, but it should report as NoMatch
-GitHubTriggersShouldReturnBothSourceRepoAndPullRequestTriggers
\ No newline at end of file
+GitHubTriggersShouldReturnBothSourceRepoAndPullRequestTriggers
+
+# ************ Below caught in $SEC101/006.GitHubPat_2021_04_05
+
+# Personal access token
+ghp_000000000001234567890123456789012345
```

Comment:
Can you please add an explicit test for this in the GitHubPatValidatorTests file? Generally, let's do this. i.e., it's fine to have these patterns in the data-driven tests. But let's add individual tests as well when working on bug fixes.

Reply:
Done.

---
In reply to: [608156134](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608156134) [](ancestors = 608156134)